jobs/bump-lockfile.Jenkinsfile

```diff
@@ -47,65 +49,77 @@ try { lock(resource: "bump-${params.STREAM}") { timeout(time: 120, unit: 'MINUTE
+            // rpm-ostree issue or patch about this)
+            shwrap("cp src/config/fedora-coreos-pool.repo{,.bak}")
+            shwrap("echo cost=500 >> src/config/fedora-coreos-pool.repo")
+            shwrap("cosa fetch --strict")
+            shwrap("cp src/config/fedora-coreos-pool.repo{.bak,}")
+        }
 
-    stage("Build") {
-        shwrap("cosa build --strict")
-    }
+        stage("Build") {
+            shwrap("cosa build --strict")
+        }
 
-    fcosKola(cosaDir: env.WORKSPACE)
+        fcosKola(cosaDir: env.WORKSPACE)
 
-    stage("Build Metal") {
-        shwrap("cosa buildextend-metal")
-        shwrap("cosa buildextend-metal4k")
-    }
+        stage("Build Metal") {
+            shwrap("cosa buildextend-metal")
+            shwrap("cosa buildextend-metal4k")
+        }
 
-    stage("Build Live") {
-        shwrap("cosa buildextend-live --fast")
-        // Test metal4k with an uncompressed image and metal with a
-        // compressed one
-        shwrap("cosa compress --artifact=metal")
-    }
+        stage("Build Live") {
+            shwrap("cosa buildextend-live --fast")
+            // Test metal4k with an uncompressed image and metal with a
+            // compressed one
+            shwrap("cosa compress --artifact=metal")
+        }
 
-    try {
-        parallel metal: {
-            shwrap("kola testiso -S --scenarios pxe-install,iso-install,iso-offline-install --output-dir tmp/kola-testiso-metal")
-        }, metal4k: {
-            shwrap("kola testiso -S --scenarios iso-install,iso-offline-install --qemu-native-4k --output-dir tmp/kola-testiso-metal4k")
+        try {
+            parallel metal: {
+                shwrap("kola testiso -S --scenarios pxe-install,iso-install,iso-offline-install --output-dir tmp/kola-testiso-metal")
+            }, metal4k: {
+                shwrap("kola testiso -S --scenarios iso-install,iso-offline-install --qemu-native-4k --output-dir tmp/kola-testiso-metal4k")
+            }
+        } finally {
+            shwrap("tar -cf - tmp/kola-testiso-metal/ | xz -c9 > ${env.WORKSPACE}/kola-testiso-metal.tar.xz")
+            shwrap("tar -cf - tmp/kola-testiso-metal4k/ | xz -c9 > ${env.WORKSPACE}/kola-testiso-metal4k.tar.xz")
+            archiveArtifacts allowEmptyArchive: true, artifacts: 'kola-testiso*.tar.xz'
         }
-    } finally {
-        shwrap("tar -cf - tmp/kola-testiso-metal/ | xz -c9 > ${env.WORKSPACE}/kola-testiso-metal.tar.xz")
-        shwrap("tar -cf - tmp/kola-testiso-metal4k/ | xz -c9 > ${env.WORKSPACE}/kola-testiso-metal4k.tar.xz")
-        archiveArtifacts allowEmptyArchive: true, artifacts: 'kola-testiso*.tar.xz'
     }
 
-    // OK, it passed kola: just push to the branch. In the future, we might be
+    // OK, we're ready to push: just push to the branch. In the future, we might be
     // fancier here; e.g. if tests fail, just open a PR, or if tests passed but a
     // package was added or removed.
     stage("Push") {
```

Comment:
Should we change the commit message based on whether it was a timestamp-only change? E.g. instead of "lockfiles: bump to latest", something like "lockfiles: bump timestamp".